docs/writing-data.md

```diff
@@ -6,24 +6,89 @@ sidebar_position: 1.4
 
 This package is not capable of writing data to the blockchain, or relaying data do the blockchain. However it’s utility methods can be used to prepare data for writing to the blockchain. This will provide information that may provide guidance for doing so outside of this package.
```

Comment:
Not one of your changes but there is a tiny typo here - 'or relaying data _to_ the blockchain'